### PR TITLE
Update Camunda 8.4 Announcement For Form Linking Bug

### PR DESCRIPTION
**Pull request by berkaycanbc**
## Description

Added a caution banner due to a bug in Zeebe https://github.com/camunda/zeebe/issues/16311 and it is expected to fix in 8.4.3.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {Zeebe 8.4.3 release} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/reference/announcements.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Release date: 9th of January 2024
 
 End of maintenance: 9th of July 2025
 
+:::caution
+For existing clusters we recommend updating to `8.4.3` directly and not `8.4.0` due to issues in [Form Linking](/components/modeler/web-modeler/advanced-modeling/form-linking.md#using-the-link-button) to user tasks which can cause wrong forms linked to user tasks.
+:::
+
```
Comment (megglos) [inline docs/reference/announcements.md:16]: I would start by giving the Form Link feature context first, as for other users it doesn't matter.

```suggestion
:::caution
Customers making use of the [Form Linking](/components/modeler/web-modeler/advanced-modeling/form-linking.md#using-the-link-button) feature must update to `8.4.3` immediately as the `8.4.0`, `8.4.1` & `8.4.2` releases suffer from a [issue 16311](https://github.com/camunda/zeebe/issues/16311) that causes random user forms to get linked with any user tasks. Effectively corrupting the user ask instance and process instance affected by that bug.
:::

```

wdyt?

Comment (berkaycanbc) [reply docs/reference/announcements.md:16]: That is definitely a better one, thanks @megglos!

File: versioned_docs/version-8.4/reference/announcements.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ Release date: 9th of January 2024
 
 End of maintenance: 9th of July 2025
 
+:::caution
+For existing clusters we recommend updating to `8.4.3` directly and not `8.4.0` due to issues in [Form Linking](/components/modeler/web-modeler/advanced-modeling/form-linking.md#using-the-link-button) to user tasks which can cause wrong forms linked to user tasks.
```
Comment (megglos) [inline versioned_docs/version-8.4/reference/announcements.md:14]: need to apply the change here too

```suggestion
The [Form Linking](/components/modeler/web-modeler/advanced-modeling/form-linking.md#using-the-link-button) feature suffers from an [issue 16311](https://github.com/camunda/zeebe/issues/16311) that causes random forms to get linked with any new user task instance eventually. Effectively corrupting the user task instance and process instance affected by that bug. If you make use of that feature you must update to `8.4.3` as the `8.4.0`, `8.4.1` & `8.4.2` releases are affected by that issue.
```

**Review by megglos — APPROVED** [review]
approving, but the 8.4 content needs to be updated too

**Review by christinaausley — APPROVED** [review]

**Pull request merged**